roomserver/internal/helpers/helpers_test.go

```diff
@@ -38,7 +38,7 @@ func TestIsInvitePendingWithoutNID(t *testing.T) {
 		var authNIDs []types.EventNID
 		for _, x := range room.Events() {
 
-			roomNID, err := db.GetOrCreateRoomNID(context.Background(), x.Unwrap())
+			roomNID, roomInfo, err := db.GetOrCreateRoomNID(context.Background(), x.Unwrap())
```

Comment:
If we are returning a `roomInfo` here surely we can remove the `roomNID` as well as the info will have it?